resources/configurationFiles/analysisRNAseqGRCh38.xml

```diff
@@ -37,17 +34,20 @@
         <cvalue name='GENOME_STAR_INDEX_50' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_50bp' type='string'/>
```

Comment:
These are not yet updated for STAR 2.7.10

Reply:
Hm, but also in the hg37 file the references are all still to 2.5.2b, althought the STAR_VERSION is **only** set in the hg37 file.

1. The aligner be fixed to 2.7.10a for both hg37 and hg38?
2. Should the corresponding hg37 and hg38 files both have new indices?
3. If you want new indices, do you want to directly use the corresponding indices from refmake, or do you prefer to continue with having indices in the ngs_share directories for the default paths?

Reply:
Yes, both reference genomes could get the same STAR version and the indexes. If the refmake is producing all the files available in the `${hg38IndexDirectory}` or in ``${hg37IndexDirectory}, then we can move the default file paths to the `refmake` paths.